packages/cloog.rb

```diff
@@ -1,19 +1,43 @@
 require 'package'
 
 class Cloog < Package
-  description 'Chunky Loop Generator which is used to perform optimization in gcc'
-  homepage 'https://www.cloog.org/'
-  version '0.18.4-2'
+  description 'The CLooG Code Generator in the Polytope Model'
+  homepage 'https://github.com/periscop/cloog'
+  version '0.20.0'
+  source_url 'https://github.com/periscop/cloog/releases/download/cloog-0.20.0/cloog-0.20.0.tar.gz'
+  source_sha256 '835c49951ff57be71dcceb6234d19d2cc22a3a5df84aea0a9d9760d92166fc72'
+
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew-cross/cloog-0.18.4-2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew-cross/cloog-0.18.4-2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew-cross/cloog-0.18.4-2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew-cross/cloog-0.18.4-2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'f79bede55ba092c133a26b03c79b71a4d9e7f46c7118308f9d182f3a2ed3f2c0',
-     armv7l: 'f79bede55ba092c133a26b03c79b71a4d9e7f46c7118308f9d182f3a2ed3f2c0',
-       i686: '7bf5c2b4eb9b0d27fe10c4da4315ffb767f22dcc0281803e1f38bfbf6cbb6c74',
-     x86_64: 'e5f20db359ef15b7881f0b15f25851a462fc06ccd5e17f23cfc420a6b29b79f6',
   })
+
+  depends_on 'isl'
+  depends_on 'osl'
+
+  def self.patch
+    system "sed -i -e '293d' -e '292d' -e '76d' -e '75d' configure.ac"
+    system "sed -i -e '193d' -e '192d' -e '191d' -e '190d' -e '189d' -e '188d' Makefile.am"
```

Comment:
This line can be shorted to `system "sed -i '188,193d' Makefile.am"`

Reply:
Got it.

Reply:
Done.